examples/trials/mnist-pytorch/config_adl.yml

```diff
@@ -0,0 +1,21 @@
+authorName: default
+experimentName: example_mnist_pytorch
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+
+logCollection: http
+trainingServicePlatform: adl
+
+searchSpacePath: search_space.json
+useAnnotation: false
+tuner:
+  builtinTunerName: TPE
+  classArgs:
+    optimize_mode: maximize
+
+trial:
+  image: {replace_to_your_image_tag}  # built by ./adl.Dockerfile
```

Comment:
generally, NNI has a `msranni/nni` official image, the dependency should put into https://github.com/microsoft/nni/blob/master/deployment/docker/Dockerfile, do not need to maintain ./adl.Dockerfile again.

Reply:
sounds good. we will remove this file from PR. Then in our integration test, we will do something like what kubeflow ts does. https://github.com/microsoft/nni/blob/master/test/pipelines/pipelines-it-kubeflow.yml#L51-L54

Reply:
removed as proposed